publisher/web_utils/web_utils.py

```
@@ -0,0 +1,118 @@
+"""This module holds the WebUtils class which does all Buganizer html scraping
+  for issues under a componentid.
+"""
+import time
+from bs4 import BeautifulSoup
+from selenium import webdriver
+from message_utils import message_utils
+import constants
+
+
+class WebUtils():
+  """Responsible for all Buganizer html scraping."""
+  def __init__(self):
+    self._buganizer_issues = []
+    self._driver = None
+    self._message_util = message_utils.MessageUtils()
+
+  def setup_webdriver(self):
+    """Completes all neccessary setup for the selenium web driver.
+
+        Returns:
+            selenium.webdriver.chrome.webdriver.WebDriver: the fully loaded Chrome
+                                                          web driver with the desired profile.
+        """
+    try:
+      options = webdriver.ChromeOptions()
+      options.add_argument("user-data-dir=" + constants.PROFILE_PATH)
+      driver = webdriver.Chrome(executable_path=constants.DRIVER_PATH,
+                    options=options)
+      self._driver = driver
+      return True
+    except Exception:
```

I would prefer to raise a proper exception instead of handling it here and return true/false. You can handle the errors in main.